src/api/entities/Asset/NonFungible/Nft.ts

```diff
@@ -116,6 +116,11 @@ export class Nft extends Entity<NftUniqueIdentifiers, HumanReadable> {
       collection,
       id,
     } = this;
+
+    if (id.lte(new BigNumber(0))) {
+      return false;
+    }
+
```

Comment:
Now that we have owner storage this method should also check if the NFT has an onwer and return false if there is no owner, since that represents a redeemed/burned NFT

Reply:
I left a comment on your other PR since that has the owner method implemented